src/scenarios/init.ps1

```diff
@@ -1,24 +1,45 @@
 Param(
-    $Channel = 'master' # default is master
+    [string] $InstallDotnetFromChannel,
```

Comment:
I think you can leave this as `Channel` - lot to type otherwise? (Though autocomplete helps some.)

Reply:
Sounds right I'll keep it

Reply:
I'll still keep ```-InstallDotnetFromChannel``` as a command line argument? This better emphasizes we need to download new dotnet.

Reply:
If you think that's the better command line argument that's fine. I was suggesting that the argument could just be $Channel. Simply adding a new $Channel variable here isn't super useful.

Reply:
I'm not against using $Channel since I also assume the user knows what it means.